lib/docx/body-reader.js

```diff
@@ -178,6 +178,10 @@ function BodyReader(options) {
                     styleId: style.styleId,
                     styleName: style.name,
                     alignment: element.firstOrEmpty("w:jc").attributes["w:val"],
+                    indentLeft: element.firstOrEmpty("w:ind").attributes["w:left"],
+                    indentRight: element.firstOrEmpty("w:ind").attributes["w:right"],
+                    indentFirstLine: element.firstOrEmpty("w:ind").attributes["w:firstLine"],
+                    indentHanging: element.firstOrEmpty("w:ind").attributes["w:hanging"],
```

Comment:
It's be good to be consistent with the arguments and the final object i.e. passing in an indent object to `Paragraph`.

Reply:
Done